src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonObjectCreationHandlingAttribute.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json.Serialization;
+
+/// <summary>
+/// Determines how deserialization will handle object creation for properties.
```

Comment:
```suggestion
/// Determines how deserialization will handle object creation for fields or properties.
```

Reply:
I wish we had a better terminology for "fields or properties" but I can't really think of any. "Members" includes things like methods and events so is not specific enough. I sometimes use the term "data members" but maybe there's something more established that we could use.

cc @stephentoub @ericstj

Reply:
will be included in next operation. I agree it would be nice. I was thinking of JSON properties here.